quixstreams/models/rows.py

```diff
@@ -19,7 +19,7 @@ def __init__(
         key: Optional[Any],
         timestamp: int,
         context: MessageContext,
-        headers: Optional[MessageHeadersTuples] = None,
+        headers: KafkaHeaders = None,
```

Comment:
Same here. It's slightly counterintuitive why do I see not `Optional` (or `| None` for later pythons) but default is set to `None`.